test/e2e/performanceprofile/functests/13_llc/llc.go

```diff
@@ -782,6 +782,13 @@ var _ = Describe("[rfe_id:77446] LLC-aware cpu pinning", Label(string(label.Open
 				Expect(err).ToNot(HaveOccurred(), "Unable to fetch numa nodes")
 				coresiblings, err := nodes.GetCoreSiblings(ctx, &cnfnode)
 				Expect(err).ToNot(HaveOccurred(), "Unable to get numa information from the node")
+				getCCX = nodes.GetL3SharedCPUs(&cnfnode)
+				ccx, err := getCCX(0)
+				Expect(err).ToNot(HaveOccurred())
+				L3CacheGroupSize = ccx.Size()
+				if len(numaInfo[0]) == L3CacheGroupSize {
+					Skip("This test requires systems where L3 cache is shared amount subset of cpus")
+				}
```

Comment:
this should work, but (much) better would be to match the cpusets ID by ID

Reply:
fixed in latest commit